localization/ndt_scan_matcher/src/ndt_scan_matcher_core.cpp

```diff
@@ -280,6 +280,22 @@ void NDTScanMatcher::timer_diagnostic()
       diag_status_msg.level = diagnostic_msgs::msg::DiagnosticStatus::ERROR;
       diag_status_msg.message += "skipping_publish_num exceed limit. ";
     }
+    if (
+      state_ptr_->count("nearest_voxel_transformation_likelihood") &&
+      std::stod((*state_ptr_)["nearest_voxel_transformation_likelihood"]) < 2.3) {
+      diag_status_msg.level = diagnostic_msgs::msg::DiagnosticStatus::WARN;
+      diag_status_msg.message += "NDT score is unreliably low. ";
+    }
+    if (
+      state_ptr_->count("exe_time") && std::stod((*state_ptr_)["exe_time"]) > 13 &&
```

Comment:
(Understanding that the original code base itself does not provide the answer to the following question, but) what is the idea behind this `13` and `24`? Did you find these values empirically?

And just a minor request:: Ideally, it would be better to make these variables ROS 2 parameters, instead of hard coding. Would be glad if you could do that maybe before this PR if possible.

Reply:
Could you show us a graph or something indicating that these numbers are working well?